opentelemetry-etw-metrics/benches/exporter.rs

```diff
@@ -0,0 +1,58 @@
+//! run with `$ cargo bench --bench exporter -- --exact <test_name>` to run specific test for logs
+//! So to run test named "fibonacci" you would run `$ cargo bench --bench exporter -- --exact fibonacci`
+//! To run all tests for logs you would run `$ cargo bench --bench exporter`
+//!
+/*
+The benchmark results:
+criterion = "0.5.1"
+OS: Windows 11 Enterprise N, 23H2, Build 22631.4460
+Hardware: Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz   2.79 GHz, 16vCPUs
+RAM: 64.0 GB
+| Test                           | Average time|
+|--------------------------------|-------------|
+| exporter                       | 847.38µs    |
+*/
+
+use opentelemetry::{metrics::MeterProvider as _, KeyValue};
+use opentelemetry_etw_metrics::MetricsExporter;
+use opentelemetry_sdk::{
+    metrics::{reader::MetricReader, PeriodicReader, SdkMeterProvider},
+    runtime, Resource,
+};
+
+use criterion::{criterion_group, criterion_main, Criterion};
+
+fn export() {
```

Comment:
this is not measuring the cost of export - it is creating new runtime, create meter provider, meter, instrument, record 10,000 measurements, flush, and drop...

Reply:
Thanks for the comment. I tried to address this in my latest change.